pkgs/development/tools/godef/default.nix

```diff
@@ -2,16 +2,16 @@
 
 buildGoPackage rec {
   name = "godef-${version}";
-  version = "1.0.0";
-  rev = "7b4626be9fa8081987905fd4719d2f6628f9d8b5";
+  version = "1.1.1";
+  rev = "v${version}";
 
   goPackagePath = "github.com/rogpeppe/godef";
-  excludedPackages = "go/printer/testdata";
+  subPackages = [ "." ];
 
   src = fetchgit {
```

Comment:
Wonder why this isn't `fetchFromGitHub`.

Reply:
https://github.com/NixOS/nixpkgs/pull/55429#discussion_r255055128

Reply:
Checked the git history and it appears it was there since its inception, so I think using `fetchFromGitHub` should work.